crates/agent/tests/custom_evaluation_test.rs

```diff
@@ -0,0 +1,124 @@
+use reqwest::{Client, StatusCode};
+use serde_json::{json, Value};
+use std::env;
+
+/// Get API configuration for tests
+/// Returns (api_url, api_key) for testing
+fn get_test_api_config() -> (String, String) {
+    // Use environment variable if available, otherwise use default
+    let api_url = env::var("API_URL").unwrap_or_else(|_| "http://localhost:8000".to_string());
```

Comment:
btw, did you try fetching it directly using existing clap config?

Reply:
https://github.com/NethermindEth/wei/pull/83/commits/739162667223a56e55a164f2f603e7fd32a670ce